### PR TITLE
LP-2456 Send updated course invitation email

### PR DESCRIPTION
**Pull request by muak50**
Our course invitation email template **(COURSE_ENROLLMENT_INVITATION)** is changed, in this PR we have made changes to accommodate updated template

Files updated in this commit:

1. openedx/adg/lms/student/helpers.py: Code updated for course invitation email
2. openedx/adg/common/lib/mandrill_client/client.py: Template Id updated

### CODE REVIEW

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM 👍

File: openedx/adg/lms/student/helpers.py [hunk 1]
```diff
@@ -149,6 +150,9 @@ def compose_and_send_adg_course_enrollment_invitation_email(user_email, message_
     elif 'course' in message_context:
         message_context['course_name'] = Text(message_context['course'].display_name_with_default)
 
+    if User.objects.filter(email=user_email).exists():
+        message_context['full_name'] = User.objects.filter(email=user_email).first().profile.name
```
Comment (asadali145) [inline openedx/adg/lms/student/helpers.py:154]:
```suggestion
user = User.objects.filter(email=user_email).select_related('profile').first()
if user:
    message_context['full_name'] = user.profile.name
```

**Review by qasimgulzar — APPROVED** [review]
LGTM

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/475/workflows/workflows_id/jobs/475/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (100%)
openedx/adg/lms/student/helpers.py (25.0%): Missing lines 153-155
-------------
Total:   5 lines
Missing: 3 lines
Coverage: 40%
-------------
```

**Review by asadali145 — APPROVED** [review]

**Comment by muak50** [comment]
> ## Diff Coverage Report - ADG
> 
> Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/475/workflows/workflows_id/jobs/475/artifacts)**. Coverage !!! 😱
> 
> ```
> -------------
> Diff Coverage
> Diff: origin/develop...HEAD, staged and unstaged changes
> -------------
> openedx/adg/common/lib/mandrill_client/client.py (100%)
> openedx/adg/lms/student/helpers.py (25.0%): Missing lines 153-155
> -------------
> Total:   5 lines
> Missing: 3 lines
> Coverage: 40%
> -------------
> ```

We will be writing tests for this in separate story.. We already made a [ticket](https://philanthropyu.atlassian.net/browse/LP-2459) for that

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - Edx
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/473/workflows/workflows_id/jobs/473/artifacts)**. Coverage !!! :scream:
```
-------------
Diff Coverage
Diff: origin/master...HEAD, staged and unstaged changes
-------------
openedx/adg/common/lib/mandrill_client/client.py (100%)
openedx/adg/lms/student/helpers.py (33.3%): Missing lines 153-154
-------------
Total:   4 lines
Missing: 2 lines
Coverage: 50%
-------------
```

**Pull request merged**